data/actions/scripts/other/create_bread.lua

```diff
@@ -4,13 +4,13 @@ local millstones = {1381, 1382, 1383, 1384}
 function onUse(player, item, fromPosition, target, toPosition, isHotkey)
 	local itemId = item:getId()
 	if itemId == 2692 then
-		if target.type == 1 and table.contains(liquidContainers, target.itemid) then
+		if target.type == 1 and table.contains(liquidContainers, target:getId()) then
```

Comment:
sorry, I completely forgot about `.type` to `getSubType()`

Reply:
target:getSubType() does not work, because getSubType() is for items.
I need to do a review on getUniqueId(). Wait.

Reply:
I think somehow, .type sometimes is used as getCount?